serving/docker/lmi-container-requirements-common.txt

```diff
@@ -23,10 +23,11 @@ onnx
 sentence_transformers
 onnxruntime-gpu==1.20.0
 autoawq==0.2.5
-tokenizers==0.20.3
```

Comment:
any reason this needs to be changed? this would be a downgrade from 0.20.3 to 0.20.0 right?

Reply:
This change was from @shen. Let me test with 0.20.3